file.go

```diff
@@ -1,37 +1,140 @@
 package jcsv
 
-import "os"
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	data []map[string]interface{} //a 1D-array of maps having string keys and empty interface values
```

Comment:
```suggestion
	// to store parsed data
	data []map[string]interface{}
```

Reply:
Sure Faizan bhai, I will follow these paradigms of commenting and writing code.